RickLeinecker/Fortuna
# Added horizontal scrollbar to Casus

**Description**
Added a horizontal scrollbar to Casus, and fixed a bug with the block bank not immediately rerendering when the window was resized.

Resolves #214 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.
Passes flow:
![image](https://user-images.githubusercontent.com/18317476/77196020-fa0e7680-6ab8-11ea-9724-aa4bb662f57d.png)

Here's the scrollbar:
![image](https://user-images.githubusercontent.com/18317476/77196090-15798180-6ab9-11ea-93b5-95440d407e50.png)


**Final Checklist:**
Go down the list and check them off.
